SoftLayer/CLI/vlan/edit.py

```diff
@@ -22,6 +23,9 @@
 def cli(env, identifier, name, note, tags):
     """Edit a vlan's details."""
 
+    if not (name or note or tags):
```

Comment:
For more than 2 options, I prefer using [`any()`](https://docs.python.org/3/library/functions.html#any)  as I feel it is a little easier to read.
```suggestion
    if not any([name, note, tags]):
```

Reply:
Done